dlt/common/destination/reference.py

```diff
@@ -185,6 +196,13 @@ def __init__(self, schema: Schema, config: DestinationClientConfiguration) -> No
         self.schema = schema
         self.config = config
 
+    @property
+    def dataset_name(self) -> str:
+        return None
+
+    def drop_dataset(self) -> None:
```

Comment:
we should not need that... and if anything you can rename it to `drop_storage`. better let's discuss it

Reply:
up to you, we can rename it, I just needed a unified interface in the tests for sql and non sql clients.